src/components/GraphFilter/DurationDropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { DropdownButton, MenuItem } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+type DurationDropdownProps = {
+  disabled: boolean;
+  initialDuration: number;
+  onClick: PropTypes.func;
+};
+
+type DurationDropdownState = {
+  currentDuration: number;
+};
+
+export class DurationDropdown extends React.Component<DurationDropdownProps, DurationDropdownState> {
+  static DurationIntervals = [
+    [60, '1 minute'],
+    [600, '10 minutes'],
+    [1800, '30 minutes'],
+    [3600, '1 hour'],
+    [14000, '4 hours'],
```

Comment:
that's 3 hours and 53 minutes :)
14400 = 4 hours

Reply:
!!! Wow Nice Thanks ^^